lib/clientBuilder.js

```diff
@@ -166,6 +166,17 @@ proto.verifyRecoveryToken = function (opts) {
   return tx.postToTransaction(this, '/api/v1/authn/recovery/token', opts);
 };
 
+// { resource, (rel), (requestContext)}
+proto.webfinger = function (opts) {
+  var url = '/.well-known/webfinger' + util.toQueryParams(opts);
+  var options = {
+    headers: {
+      'Accept': 'application/jrd+json'
+    }
+  };
+  return tx.getFromTransaction(this, url, options);
```

Comment:
This isn't a transaction (transactions are only created by the `/authn` api), so we should make the call directly:

```javascript
return http.get(sdk, url);
```

Reply:
👍 Got it!